engine/rpcserver.go

```diff
@@ -696,14 +696,14 @@ func (s *RPCServer) GetAccountInfoStream(r *gctrpc.GetAccountInfoRequest, stream
 	}()
 
 	for {
-		data, ok := <-pipe.Channel()
+		_, ok := <-pipe.Channel()
 		if !ok {
 			return errDispatchSystem
 		}
```

Comment:
Potential leak on channel when there is no data incoming. 
```suggestion
				select {
		case <-stream.Context().Done():
			return stream.Context().Err()
		case _, ok := <-pipe.Channel():
			if !ok {
				return errDispatchSystem
			}
		}
```